gensim/models/phrases.py

```diff
@@ -171,17 +175,18 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
 
         """
         if min_count <= 0:
-            raise ValueError("min_count should be at least 1")
+            min_count = 1
 
         if threshold <= 0:
             raise ValueError("threshold should be positive")
 
+        self.recode_to_utf8 = recode_to_utf8
         self.min_count = min_count
         self.threshold = threshold
         self.max_vocab_size = max_vocab_size
         self.vocab = defaultdict(int)  # mapping between utf8 token => its count
         self.min_reduce = 1  # ignore any tokens with count smaller than this
-        self.delimiter = delimiter
+        self.delimiter = delimiter if recode_to_utf8 else utils.any2unicode(delimiter)
```

Comment:
How does this work? Why is the delimited unicode if we're **not** recoding (using bytestrings)?

Deserves a comment at least.